reference_manual_generator.py

```diff
@@ -0,0 +1,17 @@
+from colorpair_generator import get_color_from_pairnumber
+from cable_colors import get_colorpair_names
+
+"""
+This function is implemented to create a reference manual with 
+color names and its corresponding codes for the wiring personnel
+"""
+
+
+def print_reference_manual():
+    pair_number_start_index = 1
+    pair_number_end_index = 26
```

Comment:
The reason this is 26 requires some analysis. When a color gets added, a maintainer may miss updating this. How could we make it easier to maintain?

Reply:
Yes, we should always avoid magic numbers & hard coded values. 
Logic is used now to resolve this issue "pair_number_end_index = len(color.MAJOR_COLORS) * len(color.MAJOR_COLORS) + 1"